ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/JobPartTracker.java

```diff
@@ -50,4 +50,16 @@ public JobPartTracker attachObjectCompletedListener(final ObjectCompletedListene
         }
         return this;
     }
+
+    public void removeDataTransferredListener(final DataTransferredListener listener) {
+        for (final ObjectPartTracker tracker : this.trackers.values()) {
```

Comment:
Could a JobPartTracker have both a Data and Object tracker?  And if so, do we only want to iterate trackers of a certain type?  Not sure I'm following what's going on here clearly.